test/prism_regression/ensure.rb

```diff
@@ -7,6 +7,24 @@
   bar
 end
 
+# Testing an ensure clause in a method definition
+def method_with_ensure
+  method_1
+  method_2
+ensure
+  ensured_method
+end
```

Comment:
Can you also test the statements being empty? I think that might lead to `stmts` being `null`

```ruby
def method_with_ensure
ensure
  ensured_method
end
```